src/modules/utils/Helper.ts

```diff
@@ -152,13 +153,10 @@ export const getUserOpHash = (
   return keccak256(enc)
 }
 
-export const getRandomSigner = (): SignerData => {
+export const getRandomSigner = (): PrivateKeyAccount => {
```

Comment:
?
cc @joepegler

Reply:
Changed the type to return the whole account object, gives us more control and more utility

Reply:
is this a custom type as well?